### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by RicardoEgas**
We checked the accessibility for this project webpage and changed the following to meet the requirements:

- Page titles - issue found:

[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L8](url)

- [Image text alternatives** (alt text)] - issues found
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L54](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L64](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L80](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L81](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L90](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L94-L97](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L108](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L121](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L133](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L144](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L157](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L169](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L199](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L207](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L208](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L223-L227](url)

- [Text headings] - no issue;

- [Color contrast] - no issue;

- [Resize] - no issue;

- [Interaction] - issues found
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L233](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L238](url)
[https://github.com/RicardoEgas/MyPortfolio/blob/8a04195bcae5443cb875a5a5af6436e2efed9cc0/index.html#L243](url)

- [Moving content] - no issue;

- [Multimedia] - no issue;

- [The basic structure of the page] - no issue;

### CODE REVIEW

File: index.html
Comment (AbelG101) [inline index.html:1]: - Great job determining changes in your Portfolio 👏 however you need to make some changes as requested in the project requirement.
    - [ ] As specified in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md) you have to make a report on your PR description listing all the points you have covered. Check out the example below:
    - ```
      ...
      Page titles - no issues found
      Image text alternatives - issues found
      [link to the corresponding line of code/s]
      ...
       ```
     - [ ] Once again as specified in the project requirements please make sure to link to the corresponding line of code ([here's how](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148)) where the changes are required in your PR description. 
     ![firefox_Qld0tf4cYR](https://user-images.githubusercontent.com/104968236/231188168-e3028e71-c243-4c39-894d-22e66ec8cc48.png)

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @RicardoEgas 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- You just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @RicardoEgas, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @RicardoEgas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**